minor edits, typo fixes

aecary opened the pull request.

aecary commented:
Alex: I'm read to pass this chapter on. There are some things left unfinished that it seemed better to let you tackle. I've left <!--TODO...--> where this is the case. It mainly involves diagrams/visuals that you will be much better suited to clean up, as well as some additional exercises that are needed (most are WeBWorK; I did the static ones).

aecary commented:
Two make check questions:

1. Regarding autoname="title" within an xref, is this a true issue? I can't remember if I should leave these despite make check errors or change them.
2. Regarding the make check errors for xref inside <intertext>, should these be removed? Or re-written so that it's inside a paragraph outside of <md> instead of in <intertext>?

Alex-Jordan commented on src/section-exponent-rules-and-scientific-notation.ptx:
```diff
@@ -37,200 +37,40 @@
-        <paragraphs>
-            <title>Product to a Power Rule</title>
-            <p>When you have a product raised to an exponent, you apply the exponent to each factor in the product. <me>\left(x\cdot y\right)^n = x^{n}\cdot y^{n}</me></p>   
+        <p>In the section <xref ref="section-introduction-to-exponent-rules">Section</xref>, we introduced three basic rules involving products and exponents.  We'll begin with a brief recap and explanation of these three exponent rules for multiplication.</p>
```
While this is still open, want to change `In the section <xref ref="section-introduction-to-exponent-rules">Section</xref>` to `In <xref ref="section-introduction-to-exponent-rules">Section</xref>`?

Alex-Jordan commented on src/section-exponent-rules-and-scientific-notation.ptx:
```diff
@@ -37,200 +37,40 @@
-
+ -->
+ <!--TODO: ask Alex how to display steps 1 and 3 correctly-->
         <exercise>
```
Just a general FYI, `webwork` elements will no longer need those `var/static` things inside the `setup` element. This is/was for making print. But the new scheme will get those values from the WeBWorK server. It's not in `mathbook` yet, but it is all working on my computer and it's what I used to make the pdf we have this term. (My point is, it is harmless to leave these out or delete them, and you definitely shouldn't spend any time maintaining them.)

Alex-Jordan commented on src/section-exponent-rules-and-scientific-notation.ptx:
```diff
@@ -37,200 +37,40 @@
         </exercise>
-
+ -->
+ <!--TODO: ask Alex how to display steps 1 and 3 correctly-->
```
Is it possible to describe here what is not being displayed quickly? Just wondering if I should pull and compile to see.

Alex-Jordan replied:
OK, change things like `$step1 = Compute("$var**($a*$b)*$var**($c*$d)");` to just make a string for LaTeX. Using double quotes means the perl variables will be replaced with their values:`$step1 = "$var^{$a \cdot $b} \cdot $var^{$c \cdot $d}";`.

Alex-Jordan commented on src/section-exponent-rules-and-scientific-notation.ptx:
```diff
@@ -311,52 +161,68 @@
 
-        <p>The previous example shows how the <term>Quotient to a Power Rule</term> works. For real numbers <m>a</m> and <m>b</m>, with <m>b \neq 0</m>:<me>\left( \frac{a}{b}  \right)^{n} = \frac{a^{n}}{b^{n}}</me>When you have a quotient raised to an exponent, you apply the exponent to the entire numerator and denominator.</p>
+        <p>The previous example shows how the <term>quotient to a power rule</term> works. For real numbers <m>a</m> and <m>b</m>, with <m>b \neq 0</m>:<me>\left( \frac{a}{b}  \right)^{n} = \frac{a^{n}}{b^{n}}</me></p>
+        <p>When you have a quotient raised to an exponent, you apply the exponent to the entire numerator and denominator. This means that we can directly compute <m>\left(\dfrac{7}{y}\right)^4</m>:
```
Thinking out loud, not necessarily making a suggestion. Do we want to mix vocabulary terms from {"fraction", "numerator", "denominator"} with vocabulary terms from {"quotient", "dividend", "divisor"}?

Alex-Jordan commented on src/section-graphing-quadratic-functions.ptx:
```diff
@@ -232,9 +232,9 @@
-
+	<p>As we can see in <xref ref="table-quadratic-function-open-upward">Table</xref> and <xref ref="table-quadratic-function-open-downward">Table</xref>, for large positive values of <m>x</m> and for large negative values of <m>x</m> the <m>x^2</m> term dominates. This is why its coefficient will determine whether the parabola opens upward or downward.</p>
 
+<!--TODO: code this to accept both "up" and "upward"; both "down" and "downward"-->
```
```
Context()->strings->add(
'up'=>{alias=>'upward'}, 
'upwards'=>{alias=>'upward'}, 
'down'=>{alias=>'downward'}, 
'downwards'=>{alias=>'downward'},
);

aecary commented:
I fixed this. Can you confirm that this is how links should be stated, as
opposed to <xref ref="---" autoname="title"/>?

*Ann Cary*
(she/her pronouns)
Mathematics Instructor
Curriculum Committee Chair
Portland Community College
Sylvania Campus, ST 203
971-722-7590
<http://goog_233805033>
http://spot.pcc.edu/~acary/



On Sat, Oct 28, 2017 at 8:26 PM, Alex Jordan <notifications@github.com>
wrote:

> *@Alex-Jordan* commented on this pull request.
> ------------------------------
>
> In src/section-exponent-rules-and-scientific-notation.ptx
> <https://github.com/PCCMathSAC/orcca/pull/197#discussion_r147569741>:
>
> > -        <paragraphs>
> -            <title>Product Rule</title>
> -            <p>When you're multiplying two expressions that have the same base, you can simplify the product by adding the exponents. <me>
> -            x^m \cdot x^n = x^{m+n}</me></p>
> -        </paragraphs>
> -
> -
> -        <paragraphs>
> -            <title>Power to a Power Rule</title>
> -            <p>When you have a base raised to an exponent and that expression is raised to another exponent, you multiply the exponents. <me>\left(x^m\right)^n = x^{m \cdot n}</me></p>
> -        </paragraphs>
> -
> -        <paragraphs>
> -            <title>Product to a Power Rule</title>
> -            <p>When you have a product raised to an exponent, you apply the exponent to each factor in the product. <me>\left(x\cdot y\right)^n = x^{n}\cdot y^{n}</me></p>
> +        <p>In the section <xref ref="section-introduction-to-exponent-rules">Section</xref>, we introduced three basic rules involving products and exponents.  We'll begin with a brief recap and explanation of these three exponent rules for multiplication.</p>
>
> While this is still open, want to change In the section <xref
> ref="section-introduction-to-exponent-rules">Section</xref> to In <xref
> ref="section-introduction-to-exponent-rules">Section</xref>?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PCCMathSAC/orcca/pull/197#pullrequestreview-72686483>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGBwAxxfIffS8VuTdkFK6ZJNTJTc9mb-ks5sw_BrgaJpZM4QJ8On>
> .
>

aecary commented:
Sure--since this is where the rules are being covered, I wanted to see
y^(2*2) before y^4. Similarly, I wanted to see y^(4+15) before y^19. Really
intermediate steps, but done so as to highlight precisely what rules are
being applied (and how).

*Ann Cary*
(she/her pronouns)
Mathematics Instructor
Curriculum Committee Chair
Portland Community College
Sylvania Campus, ST 203
971-722-7590
<http://goog_233805033>
http://spot.pcc.edu/~acary/



On Sat, Oct 28, 2017 at 8:31 PM, Alex Jordan <notifications@github.com>
wrote:

> *@Alex-Jordan* commented on this pull request.
> ------------------------------
>
> In src/section-exponent-rules-and-scientific-notation.ptx
> <https://github.com/PCCMathSAC/orcca/pull/197#discussion_r147569777>:
>
> >              <webwork source="BasicAlgebra/Exponents/exponentsMultiplication25.pg" />
>          </exercise>
> -
> + -->
> + <!--TODO: ask Alex how to display steps 1 and 3 correctly-->
>
> Is it possible to describe here what is not being displayed quickly? Just
> wondering if I should pull and compile to see.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PCCMathSAC/orcca/pull/197#pullrequestreview-72686530>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGBwAzxBKNng_9_CqMzD5p4t8RgVXzSLks5sw_GDgaJpZM4QJ8On>
> .
>

Alex-Jordan commented:
Yes, it's good now.

FYI, `autoname='title'` is now a thing of the past. To achieve what that was used for, now use `text='title'`. This is for when you want to cross-reference something and the clickable thing is just the title of the target, with no numbering. But here I think we'd want the clickable to be "Section 9.2" (or whatever the number is).

aecary commented:
As long as it's not confusing, I don't have an issue with it. I see your
point though and will mull it over if there's a way it could be phrased
better.

*Ann Cary*
(she/her pronouns)
Mathematics Instructor
Curriculum Committee Chair
Portland Community College
Sylvania Campus, ST 203
971-722-7590
<http://goog_233805033>
http://spot.pcc.edu/~acary/



On Sat, Oct 28, 2017 at 8:41 PM, Alex Jordan <notifications@github.com>
wrote:

> *@Alex-Jordan* commented on this pull request.
> ------------------------------
>
> In src/section-exponent-rules-and-scientific-notation.ptx
> <https://github.com/PCCMathSAC/orcca/pull/197#discussion_r147569880>:
>
> >                  </md>Similar to the product to a power rule, we essentially applied the outer exponent to the entire numerator and denominator. </p>
>              </solution>
>          </example>
>
> -        <p>The previous example shows how the <term>Quotient to a Power Rule</term> works. For real numbers <m>a</m> and <m>b</m>, with <m>b \neq 0</m>:<me>\left( \frac{a}{b}  \right)^{n} = \frac{a^{n}}{b^{n}}</me>When you have a quotient raised to an exponent, you apply the exponent to the entire numerator and denominator.</p>
> +        <p>The previous example shows how the <term>quotient to a power rule</term> works. For real numbers <m>a</m> and <m>b</m>, with <m>b \neq 0</m>:<me>\left( \frac{a}{b}  \right)^{n} = \frac{a^{n}}{b^{n}}</me></p>
> +        <p>When you have a quotient raised to an exponent, you apply the exponent to the entire numerator and denominator. This means that we can directly compute <m>\left(\dfrac{7}{y}\right)^4</m>:
>
> Thinking out loud, not necessarily making a suggestion. Do we want to mix
> vocabulary terms from {"fraction", "numerator", "denominator"} with
> vocabulary terms from {"quotient", "dividend", "divisor"}?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PCCMathSAC/orcca/pull/197#pullrequestreview-72686622>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGBwA2s89y3HtEsOKhTWrkqdTv8WwTpjks5sw_P0gaJpZM4QJ8On>
> .
>

aecary commented:
That wording might be rough in some instances, and I'll come back to it-- I'm going to re-read and edit how those appear with fresh eyes in the morning.

Alex-Jordan commented:
I think we probably don't want things like "Property 7.2" (or "Rule 7.2")
showing up as cross-reference clickables when the target it something
special like these exponent rules. This is unlike cross-references to
chapters or sections, which I think are OK to come out as "Section 7.2".
And also unlike cross-references to more "generic" things like "Example
7.2".

My best idea at the moment is that somewhere there is:
```
<fact xml:id="fact-exponent-product-rule">
   <title>The Exponent Product Rule</title>
   ...
```
I'm not saying it has to be a "fact", just using that as an example. I'm
also not saying anything specific about how to title these things in our
book, *except* I might be saying that the word "The" should be part of the
title.

Then, later you have
```
... according to <xref ref="fact-exponent-product-rule" text="title" /> ...
```

The `text='title'` is replacing the old `autoname='title'`. If we omit the
word "The" from the actual `title`, or do not want it capitalized in the
clickable, then I think the following works but is not tested:
```
... according to <xref ref="fact-exponent-product-rule" text="title">the
exponent product rule</xref> ...
```
I think (but have not tested) that `text='title'` prevents a number from
being used. And then presence of some content in the `xref` element
overrules the actual title.









On Sat, Oct 28, 2017 at 9:13 PM, aecary <notifications@github.com> wrote:

> That wording might be rough in some instances, and I'll come back to it--
> I'm going to re-read and edit how those appear with fresh eyes in the
> morning.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/PCCMathSAC/orcca/pull/197#issuecomment-340236444>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AEg3AArBVgOKFlbusIvkRrx6zmANfggkks5sw_tOgaJpZM4QJ8On>
> .
>



-- 
Alex Jordan
Mathematics Instructor
Portland Community College

aecary commented:
Ok, I'll look closer at those tomorrow.

I do think that it's odd to call the "power to a product" exponent rule a
"Property" or a "Rule", or a "Fact." (Although it could techincally be any
of those). Having it appear as the "power to product rule" (no caps) *and*
hyperlinked makes the most sense to me.

*Ann Cary*
(she/her pronouns)
Mathematics Instructor
Curriculum Committee Chair
Portland Community College
Sylvania Campus, ST 203
971-722-7590
<http://goog_233805033>
http://spot.pcc.edu/~acary/



On Sat, Oct 28, 2017 at 9:31 PM, Alex Jordan <notifications@github.com>
wrote:

> I think we probably don't want things like "Property 7.2" (or "Rule 7.2")
> showing up as cross-reference clickables when the target it something
> special like these exponent rules. This is unlike cross-references to
> chapters or sections, which I think are OK to come out as "Section 7.2".
> And also unlike cross-references to more "generic" things like "Example
> 7.2".
>
> My best idea at the moment is that somewhere there is:
> ```
> <fact xml:id="fact-exponent-product-rule">
> <title>The Exponent Product Rule</title>
> ...
> ```
> I'm not saying it has to be a "fact", just using that as an example. I'm
> also not saying anything specific about how to title these things in our
> book, *except* I might be saying that the word "The" should be part of the
> title.
>
> Then, later you have
> ```
> ... according to <xref ref="fact-exponent-product-rule" text="title" />
> ...
> ```
>
> The `text='title'` is replacing the old `autoname='title'`. If we omit the
> word "The" from the actual `title`, or do not want it capitalized in the
> clickable, then I think the following works but is not tested:
> ```
> ... according to <xref ref="fact-exponent-product-rule" text="title">the
> exponent product rule</xref> ...
> ```
> I think (but have not tested) that `text='title'` prevents a number from
> being used. And then presence of some content in the `xref` element
> overrules the actual title.
>
>
>
>
>
>
>
>
>
> On Sat, Oct 28, 2017 at 9:13 PM, aecary <notifications@github.com> wrote:
>
> > That wording might be rough in some instances, and I'll come back to it--
> > I'm going to re-read and edit how those appear with fresh eyes in the
> > morning.
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/PCCMathSAC/orcca/pull/197#issuecomment-340236444>,
> or mute
> > the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AEg3AArBVgOKFlbusIvkRrx6zmANfggkks5sw_tOgaJpZM4QJ8On>
> > .
> >
>
>
>
> --
> Alex Jordan
> Mathematics Instructor
> Portland Community College
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PCCMathSAC/orcca/pull/197#issuecomment-340237002>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AGBwA7pZlIugr13f7dU1bdY7nOLW7Poiks5sw_-igaJpZM4QJ8On>
> .
>
